nasa-gibs/worldview
# WV-3375: Update LANCE acronym, remove www from earthdata web address

## Description

Fixes # WV-3375

Update LANCE acronym, remove www from earthdata web address

## How To Test

1. Go to the About page
2. Find the section about LANCE, see that it's been updated to "Earth observation" from "Eo"
3. Check to see links that were www.earthdata.nasa.gov in layer descriptions now work as earthdata.nasa.gov, except for https://www.earthdata.nasa.gov/apt/documents/chlor-a/v1.0 which does not work without www.

@nasa-gibs/worldview
